src/current/v23.1/backup-architecture.md

```diff
@@ -120,7 +120,7 @@ During a [restore]({% link {{ page.version.version }}/restore.md %}) job, the jo
 
 #### Job coordination on Serverless clusters
```

Comment:
Should `Serverless` be replaced in this header?

Reply:
Yes, Done!